packages/rainbowkit/src/index.ts

```diff
@@ -1,6 +1,7 @@
 export * from './components';
 export { wallet } from './wallets/walletConnectors';
 export { getDefaultWallets } from './wallets/getDefaultWallets';
+export { getWalletConnectConnector } from './wallets/getWalletConnectConnector';
```

Comment:
This is exposed so consumers can create custom WalletConnect wallets while sharing the same connector instance. I've updated the docs to reflect this.